build-system/tasks/performance/test-suite.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const argv = require('minimist')(process.argv.slice(2));
+const loadConfig = require('./load-config');
+const {expect} = require('chai');
+const {getReport} = require('./print-report');
+// Set threshold to 1.05 by default
+const THRESHOLD = argv.threshold ? argv.threshold + 1 : 1.05;
```

Comment:
Maybe call this `DEFAULT_THRESHOLD`?

Reply:
It's not the default though?

Reply:
Yes, what you said 😃 (The intent of my comment was to move the default to its own const)

Reply:
Ah ok! Done